pkg/executor/local.go

```diff
@@ -31,7 +31,7 @@ import (
 // on local machine via exec.Command.
 // It runs command as current user.
 type Local struct {
-	commandDecorators isolation.Decorator
+	commandDecorators isolation.Decorators
```

Comment:
Now we can give multiple isolators via elipsis (`...`) in `NewLocalIsolated(decorators ...isolation.Decorators)`.